azure/README.md

```diff
@@ -0,0 +1,104 @@
+# Pytorch Elastic on Azure
+This directory contains scripts and libraries that help users run pytorch elastic jobs on Azure.
+
+## Prerequisites
+1. Familiarity with [Azure](https://azure.microsoft.com/en-us/), [aks-engine](https://github.com/Azure/aks-engine), [Azure Blob Storage](https://azure.microsoft.com/en-us/services/storage/blobs/)
+2. Quota available for Standard_DS1_v2 instance and Standard_NC6s_v3 instance.
+3. Access to Azure subscription, Resource Group and Storage. (Refer [here](https://github.com/Azure/MachineLearningNotebooks/blob/master/configuration.ipynb) for setup instructions).
+
+# Sample Usage
+
+1. #### Configure your job yaml and kubernetes.json
+```
+python petctl.py configure --name "test_job" --min_size 1 --max_size 5
```

Comment:
Is there a way to build this functionality into the existing `petctl` script for AWS? It would be great to not have one control script per cloud so they all do not diverge over time.

Reply:
Existing petctl script under AWS directory is customized for AWS resources. Using the same script for both Azure and AWS would require a lot of special case handling within the script. I agree that a single control script independent of cloud provider would present a better user experience. In my opinion there should be a single control interface that cloud providers can implement for their resources. This would ensure that implementation specifics are abstracted and provides a unified experience for end user.

Reply:
Thanks for the input. I'll work on a common set of verbs so each cloud-specific implementation does roughly the same thing. We can go ahead with your current approach for now.